src/main/java/org/zalando/nakadi/domain/EventTypeSchema.java

```diff
@@ -37,6 +46,24 @@ public void setSchema(final String schema) {
         this.schema = schema;
     }
 
+    @Nullable
```

Comment:
@v-stepanov I tried to explain in the PR description. EventTypeSchema might have `null` `version` and `createdAt` fields when updating. If a user is publishing a new schema, it doesn't have a version yet, because Nakadi will automatically generate such version for the user.

So, even though getting this attribute might lead to `null`, hence the `@Nullable`, it should never be so in actual usages, because we would have set it to some value automatically. That's what I mean by if a null shows up, then it's a bug.

It doesn't make sense to treat "nulls" since that would be silently covering a bug (if we failed to automatically set the version, for example).